external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -110,6 +110,34 @@ private[kafka010] class KafkaOffsetReader(
     kafkaReaderThread.shutdown()
   }
 
+  /**
+   * Commits offsets to the consumer_offsets topic
+   * for the driver consumer.
+  */
+  def commitOffsetsAsync(
+      partitionToOffsets: Map[TopicPartition, Long],
+      s: KafkaCommitsSource)
+    : Unit = {
+    if (_consumer != null) runUninterruptibly {
```

Comment:
Why not `consumer.`?
This way the code can be like `Unit = runUninterruptibly {`.

Reply:
I just re-use the existing variable as in `close` method. Basically the consume method will try to initialize it and here we assume there is a consume of not something wrong happened we dont want to initialize it.